product_docs/docs/pgd/5/known_issues.mdx

```diff
@@ -2,56 +2,51 @@
-    - Using the GUC `bdr.writers_per_subscription`. See [Configuration of generic replication](/pgd/latest/reference/pgd-settings/#generic-replication).
-
--   There currently is no protection against altering or removing a commit scope. Running transactions in a commit scope that's concurrently being altered or removed can lead to the transaction blocking or replication stalling completely due to an error on the downstream node attempting to apply the transaction. Ensure that any transactions using a specific commit scope have finished before altering or removing it.
-
--   The [PGD CLI](cli) can return stale data on the state of the cluster if it's still connecting to nodes that were previously parted from the cluster. Edit the [`pgd-cli-config.yml`](cli/configuring_cli/#using-a-configuration-file) file, or change your [`--dsn`](cli/configuring_cli/#using-database-connection-strings-in-the-command-line) settings to ensure only active nodes in the cluster are listed for connection.
-
-Details of other design or implementation [limitations](limitations) are also available.
+This section discusses currently known issues in EDB Postgres Distributed 5.
+These known issues are tracked in PGD's ticketing system and are expected to be
+resolved in a future release.
+
+-   The Decoding Worker feature doesn't work with CAMO/EAGER/Group Commit.
+    Installations using CAMO/Eager/Group Commit must keep `enable_wal_decoder`
+    disabled.
+
+-   Lag Control doesn't adjust commit delay in any way on a fully isolated node,
+    that is, in case all other nodes are unreachable or not operational. As soon
+    as at least one node is connected, replication Lag Control picks up its work
+    and adjusts the PGD commit delay again.
+
+- Only one kind of `GROUP COMMIT` `commit_decision` setting can be used in a
+  single commit scope. Where multiple `GROUP COMMIT` clauses are used in a
+  commit scope, they must have the same `commit_decision` setting (`async` or
+  `eager`). As CAMO is effectively an alias of `GROUP COMMIT` (with
+  `commit_decision` set to `partner`), you cannot mix GROUP COMMITs with CAMO.
+
+- Eager Replication currently only works by using the "global" PGD commit scope.
+
+- Transactions using Eager Replication can't yet execute DDL. The TRUNCATE command is allowed.
+
+-   Parallel apply is not currently supported in combination with Group Commit or Eager,
+    please make sure to disable it when using Group Commit by either setting
+    `num_writers` to 1 for the node group (using
+    [`bdr.alter_node_group_config`](/pgd/latest/reference/nodes-management-interfaces/#bdralter_node_group_config))
+    or via the GUC `bdr.writers_per_subscription` (see [Configuration of Generic
+    Replication](/pgd/latest/reference/pgd-settings/#generic-replication)).
+    (Parallel Apply with CAMO is supported from PGD 5.2 onwards.)
```

Comment:
:+1:  .. link to BDR-3308, please.

Reply:
This is an external-facing document; we can't link to the internal issue tracker from it.